src/css/header.scss

```diff
@@ -0,0 +1,19 @@
+header {
+  @include size(10%, 100%);
+  background-color: $opaque-white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Issues here. The space-around looks much better, but it's not what I really want.